backend/src/utils/tokens.ts

```
@@ -0,0 +1,80 @@
+
+/**
+ * Generate refresh token
+ */
+export const generateRefreshToken = (): string => {
+  return generateSecureToken(64);
+};
+
+/**
+ * Generate JWT access token
+ */
+export const generateAccessToken = (payload: any): string => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)